src/game/boe.menu.hpp

```diff
@@ -10,14 +10,16 @@
 #include "boe.consts.hpp"
 #include "spell.hpp"
 #include "skills_traits.hpp"
+#include "event_listener.hpp"
+#include "drawable.hpp"
 
-class OpenBoEMenu {
+class OpenBoEMenu : public iEventListener, public iDrawable {
 public:
 
 	OpenBoEMenu(sf::RenderWindow&, cUniverse&);
 
-	bool handle_event(const sf::Event&);
-	void draw();
+	virtual bool handle_event(const sf::Event&) override;
```

Comment:
You don't actually _need_ the virtual keyword here as the virtual property is inherited... not that it hurts or anything.

Reply:
I will leave this as is.